src/app/dashboard/dashboard.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="dashboard-screen">
+
+  <!-- <h3 class="dashboardTitle">Dashboard</h3> -->
+
+  <div class="user-data">
+      <h5>Your profile:</h5>
+      <hr>
```

Comment:
use hr only when among same tags like p or img, in othewise case use css to make space